### PR TITLE
feat: Hook up granular A4 GIS API variables for Epsom and Ewell

### PR DESCRIPTION
**Pull request by augustlindemer**

### CODE REVIEW

**Comment by augustlindemer** [comment]
[Hookup Planning Data constraints for Epsom & Ewell](https://trello.com/c/3QxNSOvL/2911-hookup-planning-data-constraints-for-epsom-ewell)

File: api.planx.uk/modules/gis/service/local_authorities/metadata/epsomAndEwell
Comment (DafyddLlyr) [inline api.planx.uk/modules/gis/service/local_authorities/metadata/epsomAndEwell:1]: This file needs the TypeScript `.ts` file extension added - I think if you update this, the tests should pass 👍

File: api.planx.uk/modules/gis/service/local_authorities/metadata/epsomAndEwell.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  article4: {
+    // Planx granular values link to Digital Land article-4-direction and entity.reference
+    records: {
+      "article4.epsomAndEwell.burghheath": "11/00003/ART4",
+      "article4.epsomAndEwell.churchstreet": "11/00004/ART4",
+      "article4.epsomAndEwell.college": "11/00005/ART4",
+      "article4.epsomAndEwell.downs": "11/00006/ART4",
+      "article4.epsomAndEwell.ewellvillage": "11/00002/ART4",
+      "article4.epsomAndEwell.highergreenlongdown": "05/00002/ART4",
+      "article4.epsomAndEwell.lintons": "	11/00007/ART4",
```
Comment (jessicamcinchak) [inline api.planx.uk/modules/gis/service/local_authorities/metadata/epsomAndEwell.ts:23]: nit: looks like this one has a few some leading spaces ?

**Review by jessicamcinchak — APPROVED** [review]
Looks great, appreciate you picking these up :muscle:

**Pull request merged**